src/createServer.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { usersRouter } = require('./routes/users.routes');
+const { expensesRouter } = require('./routes/expenses.routes');
```

Comment:
You folder is named `Routes` not `routes` which is why your tests can be failed
Better to name it in lowercase